src/backend/sqlite/table.rs

```diff
@@ -144,7 +144,7 @@ impl TableBuilder for SqliteQueryBuilder {
         };
         write!(sql, "ALTER TABLE ").unwrap();
         if let Some(table) = &alter.table {
-            table.prepare(sql, self.quote());
+            table_builder::TableBuilder::prepare_table_ref(self, table, sql);
```

Comment:
Why not to use `self.prepare_table_ref`? And in other places.

Reply:
I think the compiler would be confused:

```
multiple applicable items in scope
multiple `prepare_table_ref` foundrustc[E0034](https://doc.rust-lang.org/error-index.html#E0034)
index_builder.rs(35, 5): candidate #1 is defined in an impl of the trait `backend::index_builder::IndexBuilder` for the type `sqlite::SqliteQueryBuilder`
query_builder.rs(515, 5): candidate #2 is defined in an impl of the trait `query_builder::QueryBuilder` for the type `sqlite::SqliteQueryBuilder`
```

Reply:
@nahuakang ah, yes, thank you!